modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -432,3 +432,265 @@ Status description:: error: syntax error or access rule violation - auth info ex
 === 42NFF
 
 Status description:: error: syntax error or access rule violation - permission/access denied. Access denied, see the security logs for details.
+
+== System configuration or operation exception
+
+System configuration or operation exception errors occur when there is an error in the system configuration or operation, such as procedure registration failure, a missing class field annotation, an unsupported injectable component type, duplicate field names, invalid map key type, etc.
+
+=== 51N00
+
+Status description:: error: system configuration or operation exception - procedure registration error. Failed to register procedure.
```

Comment:
```suggestion
Status description:: error: system configuration or operation exception - procedure registration error. Failed to register procedure/function.
```

This seems to have been updated recently in the code base by @anderson4j, but it still say "Failed to register procedure." in the spreadsheet. Is the code variant correct?